templates/shared/_asset-card-actions.html

```diff
@@ -1,8 +1,10 @@
 <div class="p-section--shallow">
         <button class="p-button--positive copy-link"
+                style="margin-right: 4%"
```

Comment:
On some screen sizes this doesn't work:
<img width="1334" height="861" alt="image" src="https://github.com/user-attachments/assets/a05fc017-4f3c-4c11-b33e-2eae8ec07467" />
<img width="1322" height="1067" alt="image" src="https://github.com/user-attachments/assets/de24c4c9-2444-4068-958a-4624964f1145" />